.github/workflows/ci.yml

```diff
@@ -650,6 +654,9 @@ jobs:
         run: |
           export MAVEN_OPTS="${MAVEN_INSTALL_OPTS}"
           $RETRY $MAVEN clean install ${MAVEN_FAST_INSTALL} -pl '!:trino-docs,!:trino-server-rpm'
+      - name: Download all Trino dependencies
+        run: |
+          $MAVEN dependency:go-offline
```

Comment:
Can we run this before `mvn clean install` above? Then we could add retries here and remove them for builds. We always only wanted retries to improve resiliency against network issues and it is confusing when builds fail because of compilation issues.